parser/helper/helpers.go

```diff
@@ -209,7 +215,27 @@ func (h *Helper) GetActorAddressInfo(add address.Address, key filTypes.TipSetKey
 	return addInfo
 }
 
-func (h *Helper) GetActorNameFromAddress(add address.Address, height int64, key filTypes.TipSetKey) (cid.Cid, string, error) {
+// GetActorNameFromAddress returns the actor name for the given address.
+func (h *Helper) GetActorNameFromAddress(add address.Address, height int64, key filTypes.TipSetKey) (string, error) {
+	_, actorName, err := h.GetActorInfoFromAddress(add, height, key)
+	if err != nil {
+		switch add.Protocol() {
+		// f1 or f3 is an account
+		case address.SECP256K1, address.BLS:
+			return "account", err
+		case address.Delegated:
```

Comment:
This is not for ALL delegated, but for EAM space (f410). Otherwise, we go with unknown, as it is not really correct.